boards/shields/seeed_xiao_round_display/Kconfig.defconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Nicolas Goualard <nicolas.goualard@sfr.fr>
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_SEEED_XIAO_ROUND_DISPLAY
+
+if DISPLAY
+
+if LVGL
+
+config INPUT
+	default y
+
+choice LV_COLOR_DEPTH
+	default LV_COLOR_DEPTH_16
+endchoice
+
+config LV_COLOR_16_SWAP
+	default y
+
+endif # LVGL
+endif # DISPLAY
+
+if INPUT
+config INPUT_CHSC6X
```

Comment:
You shouldn't need to enable this driver here- the Kconfig in the input subsystem should be enabled automatically by the `"chipsemi,chsc6x"` compatible in your devicetree

Reply:
Ah yup, just did that.